### ISSUE TITLE
Support for Stockcharts

### ISSUE THREAD

**Issue by ptheunis** [amcharts/animate#2]
Any possibility to get support for Stockcharts?
I was looking at the code and tried to see if i could write it myself but there are alot of assumptions about having only 1 dataset.

i'd take a pointer in the right direction at this point though. updating stockcharts on a regular interval look horribly choppy right now.

**Comment by martynasma** [comment]
I'm afraid we don't plan on implementing Animate support for Stock chart for precisely the reason - it's a much more complex chart type, which makes updating data every fraction of second not possible. Sorry.

**Comment by ptheunis** [comment]
So there is no way for me to get a smooth drawing stock graph, or a way to get auto grouping in a standard serial chart?

**Comment by martynasma** [comment]
No, I'm afraid. Unless you'd like to take a stab at modifying Animate plugin to support Stock.

Sorry.

**Comment by mg1075** [comment]
Ideally, animation for charts and stock charts would not be a plugin; instead, a built-in feature that one can pass as an option in the chart config, where default is "off" if there is a performance hit due to being "on".
